x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/connectors/connectors.spec.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { ToolingLog } from '@kbn/tooling-log';
+import { Agent } from 'supertest';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantApi');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  const CONNECTOR_API_URL = '/internal/observability_ai_assistant/connectors';
+
+  describe('List connectors', () => {
+    before(async () => {
+      await deleteAllActionConnectors(supertest);
+    });
+
+    after(async () => {
+      await deleteAllActionConnectors(supertest);
+    });
+
+    it('Returns a 2xx for enterprise license', async () => {
+      const { status } = await observabilityAIAssistantAPIClient.editor({
+        endpoint: `GET ${CONNECTOR_API_URL}`,
```

Comment:
I suggest not using variables here. It's harder to read and search for. Also, I'm not sure the typescript intellisense works with variables

```suggestion
        endpoint: `GET /internal/observability_ai_assistant/connectors`,
```

Reply:
If you really want to make it DRY I suggest adding a helper and re-use that instead:

```ts
const getConnectors = () =>
  observabilityAIAssistantAPIClient.editor({
    endpoint: `GET /internal/observability_ai_assistant/connectors`,
  });
```

Reply:
Agree. Now that I’m moving many of these tests, I won’t use a variable and will ensure they follow a consistent pattern since we currently have a mix.